secant/Util/Int64+Zcash.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  Int64+Zcash.swift
+//  secant-testnet
+//
+//  Created by Lukáš Korba on 15.04.2022.
+//
+
+import Foundation
+
+extension Int64 {
```

Comment:
Can you add a todo to improve this using Decimals and a Zatoshi type?

Reply:
fixed